modules/maven3/manifests/init.pp

```diff
@@ -5,37 +5,44 @@
 # @example
 #  include maven3
 #
-#  class { 'maven3': 
+#  class { 'maven3':
 #    version => '3.5.3',
 #  }
 
 class maven3 (
   $version = '3.5.3',
+  $load_dir = "/tmp/",
+  $source = "http://archive.apache.org/dist/maven/maven-3/${version}/binaries/",
```

Comment:
Where is our repo with mvn.
Stop wget and curl jars tar zips from the Internet use RPM.

Reply:
I'm going to use it until repo will be finished. Will be fixed in future